test/infrastructure/test-extension/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,24 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
Would be nice if we can eventually sync up config folders across CAPI & CAPV once CAPV was migrated to kustomize v5. Maintaining different variations of all of this is really a pain 
(but please not now :))

Reply:
Let's create a follow-up issue and then resolve here

Reply:
We already have that here: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2886